src/utils/GeometryRayIntersectUtilities.js

```diff
@@ -12,7 +12,7 @@ export function convertRaycastIntersect( hit, object, raycaster ) {
 	hit.distance = hit.point.distanceTo( raycaster.ray.origin );
 	hit.object = object;
 
-	if ( hit.distance < raycaster.near || hit.distance > raycaster.far ) {
+	if ( hit.distance < raycaster.near || hit.distance > raycaster.far ) { // TODO probably this can be removed
```

Comment:
This can be removed if we can guarantee the function only returns triangle hits within the range of the near and far values. I don't recall if there are any yet but if not it would be nice to add a few tests that check to make sure the correct results are returned when the raycaster near and far are modified compared to the three.js results.

Reply:
Lets go ahead and remove this now that we have tests!